website/snippets/_install-dbt-extension.md

```diff
@@ -112,64 +112,121 @@ This will start the upgrade tool and guide you through the Fusion upgrade with a
+- Registration links your editor to a dbt account so you can keep using the extension beyond the grace period.
+- This _does not_ require a <Constant name="dbt_platform" /> project — just a dbt account.
+- If a valid `dbt_cloud.yml` file exists on your machine, the extension will automatically use it and skip login.
+- If you already have a dbt account (even from years ago), you will be directed into an OAuth sign-in flow.
+
+<Expandable alt_header="Understanding regions">
+
+Most users can sign in from the extension's browser registration page for the default `US1` region. If that works, you have an account in the default region and don't need to know other [regions](/docs/cloud/about-cloud/access-regions-ip-addresses).
+
+Use a credential file (`dbt_cloud.yml`) instead of sign-in when:
+
+- You can't sign in,
+- Your organization uses a non-default region (`eu1`, `us2`, and so on), or
+- You prefer file-based credentials.
+
+If you're unsure whether you have a `US1` account from the past, try signing in or using **Forgot password** at [us1.dbt.com](http://us1.dbt.com). If nothing comes up, continue with [Register with `dbt_cloud.yml`](#register-with-dbt_cloudyml).
+</Expandable>
+
+#### Choose your registration path
+
+Your dbt VS Code extension registration path depends on your situation. Select the one that applies to you:
 
-Users must complete registration within 14 days of installing the dbt extension. There are two ways to register:
+- **New to dbt and never created a dbt account?** → Use [First-time registration](#first-time-registration)
+- **Have an existing dbt account and can sign in?** → Use [Existing dbt account](#existing-dbt-account)
+- **Email already exists / can’t sign in?** (locked, forgot password) → Use [Recover your login](#recover-your-login)
+- **Can't sign in OR your organization uses a non-default region** (`eu1`, `us2`) → Use [Register with `dbt_cloud.yml`](#register-with-dbt_cloudyml)
 
-- [Don't have a dbt account?](#new-user-registration) &mdash; Follow the VS Code [“get started”](/docs/install-dbt-extension#getting-started) workflow to register and verify your email. For the initial installation, you only need to provide your name and email address to complete the registration. Subsequent installations will require you to complete the entire [dbt account registration process](#accessing-your-dbt-account) to use the extension. 
-- [Returning user or existing <Constant name="dbt_platform" /> user?](#register-with-an-existing-dbt-account) &mdash; If you have an existing <Constant name="dbt_platform" /> or previously created one in the past (whether expired or active), no need to register! Just log in with the same email to prevent duplicate accounts. If you can't log in or if it's locked, reach out to [dbt Support](mailto:support@getdbt.com) to have it unlocked or reset before logging in.
+---
```

Comment:
We're inconsistently using these lines. Should we remove in favor of H1, H2, H3 etc?

Reply:
i see what you mean! i used the divider to try to make a clear cut/start for each h3 nested in the h2. but i can remove them!